src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -333,6 +333,17 @@ public function getCacheDir()
         return $this->getProjectDir().'/var/cache/'.$this->environment;
     }
 
+    /**
+     * Gets the build directory.
+     *
+     * @return string The build directory
```

Comment:
I would remove this phpdoc and add a type hint on the method instead.

Reply:
Oh yes good point, done!